zds/settings/abstract_base.py

```diff
@@ -1,26 +1,27 @@
-# coding: utf-8
-
-import os
-from os.path import join
+from os.path import join, dirname
+from pathlib import Path
+import toml
 
 from django.contrib.messages import constants as message_constants
 from django.utils.http import urlquote
 from django.utils.translation import gettext_lazy as _
 
-from colorlog import ColoredFormatter
 
+try:
+    secrets = toml.load('secrets.toml')
+except IOError:
+    secrets = {}
 
-# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-BASE_DIR = os.path.dirname(os.path.dirname(__file__))
 
-DEBUG = True
+# Build paths inside the project like this: join(BASE_DIR, ...)
+BASE_DIR = str(Path(__file__).parents[2])
```

Comment:
`BASE_DIR = str(Path(__file__).parent.parent.absolute())` I think. (I find it more readable

Reply:
`BASE_DIR = str(Path(__file__).parent.parent.resolve())` semble même encore un peu mieux :D

Reply:
D’ailleurs, sur d’autres projets, ils ne font même pas de `str()`. Je pense qu’on devrait se diriger vers ça à l’avenir.

Reply:
ça dépend de ce dont on a besoin. si on utilise trop les utilitaire de `os.path` on a besoin du str.